### PR TITLE
Add nlevels property for Index and MultiIndex

### PR DESCRIPTION
**Pull request by charlesdong1991**

### CODE REVIEW

File: databricks/koalas/indexes.py [hunk 1]
```diff
@@ -151,6 +151,13 @@ def names(self, names: List[Union[str, Tuple[str, ...]]]) -> None:
         names = [name if isinstance(name, tuple) else (name,) for name in names]
         self._kdf._internal = internal.copy(index_map=list(zip(internal.index_columns, names)))
 
+    @property
+    def nlevels(self):
+        """
+        Number of levels.
+        """
+        return 1
```
[the remark this answers is not in the document]
Comment (charlesdong1991) [reply databricks/koalas/indexes.py]: emm, yeah, i thought this was faster to run for Index. But yeah, I could use implementation for `MultiIndex` here.

> Also shall we add some examples?

Sure!

File: databricks/koalas/tests/test_indexes.py [hunk 2]
```diff
@@ -246,3 +246,16 @@ def test_multi_index_not_supported(self):
         with self.assertRaisesRegex(TypeError,
                                     "cannot perform all with this index type"):
             kdf.set_index(['a', 'b']).index.all()
+
+    def test_index_nlevels(self):
+        for name in ["has_name", None]:
```
[the remark this answers is not in the document]
Comment (charlesdong1991) [reply databricks/koalas/tests/test_indexes.py]: No, there is no difference regardless of having name or not, I just want to prove `nlevels` works with/without name in test. 

Do you prefer to change here? @ueshin

**Comment by HyukjinKwon** [comment]
@charlesdong1991 can you resolve conflicts? Otherwise seems we're good to go.

**Comment by charlesdong1991** [comment]
thanks! @HyukjinKwon resolved!

File: databricks/koalas/tests/test_indexes.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (HyukjinKwon) [reply databricks/koalas/tests/test_indexes.py]: Ah, yea, let's change. I think we don't have to verify the name.

[the remark this answers is not in the document]
Comment (charlesdong1991) [reply databricks/koalas/tests/test_indexes.py]: done

**Comment by HyukjinKwon** [comment]
Thanks, @charlesdong1991

**Pull request merged**